src/controllers/add.documents.controller.ts

```diff
@@ -0,0 +1,28 @@
+import { Session } from "ch-node-session-handler";
+import { NextFunction, Request, Response } from "express";
+import ObjectionCompanyProfile from "model/objection.company.profile";
+import { SESSION_COMPANY_PROFILE, SESSION_OBJECTION_ID } from "../constants";
+import { Templates } from "../model/template.paths";
+import { updateObjectionReason } from "../services/objections.service";
+import { getValidAccessToken, getValueFromObjectionsSession } from "../services/objections.session.service";
+
+const route = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
the patching should be done in the Post of enter-information screen.

Reply:
It is - see the routes.ts change below. But do you mean this file needs renaming?

Reply:
there should be a function added to the enter.information.controller to do what you've done in the add.document.controller. The patching is for the information/reason and not related to the add.documents page.

Reply:
Right. Seen your review and now see what you mean. Think I'm going to have to integrate once your PR is merged. Will approve it now.

Reply:
Updated so that implementation moved to correct enter information post function.